lca-cli/postpivot/postpivot.go

```diff
@@ -541,3 +548,31 @@ func (p *PostPivot) applyNMStateConfiguration(seedReconfiguration *clusterconfig
 
 	return nil
 }
+
+func (p *PostPivot) createPullSecretFile(seedReconfiguration *clusterconfig_api.SeedReconfiguration, pullSecretFile string) error {
+	if seedReconfiguration.PullSecret == "" {
+		p.log.Infof("Pull secret was not provided")
```

Comment:
Add TODO to make this a hard error, as seed images in the future are not going to have a pull secret and the cluster can't operate without it